### PR TITLE
Fix export for debounce, add tslint.json

### PR DESCRIPTION
**Pull request by LKay**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [X] Increase the version number in the header if appropriate.
- [X] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dslint/dt.json" }`.

### CODE REVIEW

File: types/debounce/debounce-tests.ts [hunk 1]
```diff
@@ -1,13 +1,12 @@
-
-import debounce from "debounce";
+import debounce = require("debounce");
 
 const doThings = () => 1;
 
-debounce(function(){ doThings(); })();
+debounce(() => doThings)();
```
Comment (andy-ms) [inline types/debounce/debounce-tests.ts]: This is a function returning a function, while before it was just a function.

**Review by ghost — COMMENTED** [review]
While you're here, could you implement:

> The debounced function returned also has a property 'clear' that is a function that will clear any scheduled future executions of your function.

Should be just `: A & { clear(): void; };`.

**Comment by LKay** [comment]
@andy-ms There you go!

**Review by denis-sokolov — APPROVED** [review]

**Comment by andy-ms** [comment]
Thanks!

**Pull request merged**